docs/api-reference/composite-layer.md

```diff
@@ -113,9 +113,16 @@ The default implementation returns `pickParams.info` without any change.
 
 ##### `getSubLayerProps`
 
-Returns an object with the properties of the sublayer built using the passed props and
+Parameters:
+
+* `subLayerProps` (Object)
```

Comment:
why only pass these two props? how about the other props of a certain `subLayer`?